base/LandingPage/static/CustomCSS/lower.css

```diff
@@ -1 +1,22 @@
-/* Temporarily Empty */
\ No newline at end of file
+/* Temporarily Empty */
+<!--LowerSection.html-->
+<div class="container">
+    <div class="row text-center justify-content-center">
+        {% for data in team %}
+            <!-- Modal Trigger -->
+            <!-- FIXME border being used to measure positioning of items -->
+            <div class="border col-lg-2 col-md-4 col-sm-6 col-6"
+                 data-toggle="modal"
+                 data-target="#user-modal{{ data.id }}"
+                 style="cursor: pointer;"
+            >
+                <img src="{{ data.member_photo.url }}" width="150px">
+                <h2>{{ data.member_name }}</h2>
+                <p>{{ data.member_major }}</p>
+            </div>
+        {% endfor %}
+    </div>
+</div>
+<img src="{{ data.member_photo.url }}" width="150px">
+<h2>{{ data.member_name }}</h2>
+<p>{{ data.member_major }}</p>
```

Comment:
Hey this is supposed to be for html. Please remove this section from the css file.